src/modules/launcher/Wox/ViewModel/ContextMenuItemViewModel.cs

```diff
@@ -9,11 +9,26 @@ namespace Wox.ViewModel
 {
     public class ContextMenuItemViewModel : BaseModel
     {
+        private ICommand command;
         public string PluginName { get; set; }
         public string Title { get; set; }
         public string Glyph { get; set; }
         public string FontFamily { get; set; }
-        public ICommand Command { get; set; }
+        public ICommand Command {
+            get
+            {
+                return this.command;
+            }
+
+            set
+            {
+                if (value != this.command)
+                {
+                    this.command = value;
+                    OnPropertyChanged();

```

Comment:
Can you please add test for this? We can test if the property changed is called each time `Command` is set.

Reply:
I added in a test to verify if `OnPropertyChanged` is called from the `ICommand` binding. 
However, I think it is an overkill to do it for all the occurances and to refactor the entire `ResultViewModel` just to check if the `onPropertyChanged()` function has been called because that function is being called in a lot of places in our codebase.